crypto/x509/t_x509.c

```diff
@@ -94,11 +94,11 @@ int X509_print_ex(BIO *bp, X509 *x, unsigned long nmflags,
         }
         if (l != -1) {
             if (bs->type == V_ASN1_NEG_INTEGER) {
-                l = -l;
+                l = -(unsigned long)l;
```

Comment:
I'm not sure if this is a side note (i.e. no action is required) or not. There is certain ambiguity in this. What is it it's complaining about? About long being printed as unsigned long. But does this line have to be changed? Where is it a problem to print long as unsigned? On non-two's-complement system, right? But above line would be inappropriate on non-two's-complement system, wouldn't it? Yeah, I know we support only two's-complement, but it's still kind of ambiguous approach, because we are addressing potential problems on non-two's-complements in a way that is probably inappropriate for it. ["probably" in last sentence refers to lack on hands-on experience with such systems.] BTW, it will trigger VC warning...